settings.gradle.kts

```diff
@@ -19,7 +19,7 @@ val agpWorkspaceLocationProperty = providers.gradleProperty("com.android.workspa
 val agpWorkspaceLocation  = if (agpWorkspaceLocationProperty.isPresent) {
     agpWorkspaceLocationProperty.get()
 } else {
-    System.getenv("WORKSPACE_LOCATION") ?:
+    System.getenv("AGP_WORKSPACE_LOCATION") ?:
```

Comment:
Should the other usages of WORKSPACE_LOCATION be updated too?

https://github.com/search?q=repo%3Aandroid%2Fgradle-declarative%20WORKSPACE_LOCATION&type=code

Reply:
Done !